src/base_classes/active_space_solver.h

```diff
@@ -111,6 +112,23 @@ class ActiveSpaceSolver {
         as_ints_ = as_ints;
     }
 
+    /// Save the wave function
```

Comment:
Can you say what does this function return?

Reply:
It returns a map from state to the filename of the wave function